### ISSUE TITLE
An error occurred at initialising Bisq: {0}.

### ISSUE THREAD

**Issue by Ryion32** [bisq-network/bisq2#2257]
[bisq.log](https://github.com/user-attachments/files/15569164/bisq.log)

**Comment by HenrikJannsen** [comment]
Can you post also the debug.log from the tor directory?

@alvasw 
Relevant log lines:
```
giu-04 20:03:40.656 [NetworkService.network-IO-pool-0] INFO  b.n.p.n.t.TorTransportService: Initialize Tor 
giu-04 20:03:40.675 [Thread-39] INFO  b.t.c.NativeTorController: connect with controlPort 51082 
giu-04 20:03:40.683 [Thread-39] ERROR b.t.c.NativeTorController: Connect failed using controlPort 51082 
giu-04 20:03:40.685 [NetworkService.network-IO-pool-0] ERROR b.d.DesktopApplicationService: Error at networkFuture.initialize java.util.concurrent.CompletionException: bisq.tor.controller.ControlCommandFailedException: Couldn't connect to control port.
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
```

**Comment by Ryion32** [comment]
Sure, here it is: 
[debug.log](https://github.com/user-attachments/files/15613601/debug.log)
 

However, I solved the problem by deleting the tor directory, mantaining the db directory and reinstalling bisq2. Now it works.

**Comment by Langestrand** [comment]
Is experience the same problem. I have given up on using bisq 2. 

Is the issue well known? 
Is there any guidelines to fix the problem? Or are you still working on it? 

Bg

**Comment by HenrikJannsen** [comment]
Should be fixed with version 2.1.0 (https://github.com/bisq-network/bisq2/releases/tag/v2.1.0)
